app/assets/stylesheets/components/_banner.scss

```diff
@@ -1,10 +1,43 @@
+@font-face {
+  font-family: 'Anurati-Regular';
+  src: url('../../fonts/Anurati-Regular.otf') format("truetype");
+}
+
```

Comment:
whole to whole video are for the video banner. if you dont like we can change back to just .banner